Dockerfile

```diff
@@ -15,10 +15,17 @@ RUN apt-get update && \
     # Use a fixed apt-get repo to stop intermittent failures due to flaky httpredir connections,
     # as described by Lionel Chan at http://stackoverflow.com/a/37426929/5881346
 RUN sed -i "s/httpredir.debian.org/debian.uchicago.edu/" /etc/apt/sources.list && \
-    apt-get update && apt-get install -y build-essential unzip cmake && \
-    # Work to upgrade to Python 3.7 can be found on this branch: https://github.com/Kaggle/docker-python/blob/upgrade-py37/Dockerfile
-    conda install -y python=3.6.6 && \
+    apt-get update && \
+    # Needed by vowpalwabbit & lightGBM (GPU build).
+    # https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Python#installing
+    # https://lightgbm.readthedocs.io/en/latest/GPU-Tutorial.html#build-lightgbm
+    apt-get install -y build-essential unzip cmake && \
+    apt-get install -y libboost-dev libboost-program-options-dev libboost-system-dev libboost-thread-dev libboost-math-dev libboost-test-dev libboost-python-dev libboost-filesystem-dev zlib1g-dev && \
     pip install --upgrade pip && \
+    # enum34 is a backport of the Python 3.4 enum class to Python < 3.4.
+    # No need since we are using Python 3.7. This is causing errors for packages
+    # expecting the 3.7 version of enum. e.g. AttributeError: module 'enum' has no attribute 'IntFlag'
+    pip uninstall -y enum34 && \
```

Comment:
Is the base DLVM image installing this or comes with pip upgrade?

Reply:
The DLVM image comes pre-installed with it.